osu.Game.Tests/Visual/Ranking/TestSceneAccuracyCircle.cs

```diff
@@ -24,11 +24,20 @@ namespace osu.Game.Tests.Visual.Ranking
 {
     public partial class TestSceneAccuracyCircle : OsuTestScene
     {
+        [TestCase(0, ScoreRank.D)]
         [TestCase(0.2, ScoreRank.D)]
         [TestCase(0.5, ScoreRank.D)]
+        [TestCase(0.699, ScoreRank.D)]
```

Comment:
Is there one significant digit missing here? For testing both extremities on both sides of the gap, I'd expect `0.6999` to be covered here (accuracy is generally displayed within 2 significant figures everywhere in lazer).

Reply:
Very true, I had one too few here to properly cover the case of `NOTCH_WIDTH_PERCENTAGE` being 0.002 as it is (extending 0.001 on both sides)